contentcuration/contentcuration/models.py

```diff
@@ -90,9 +90,9 @@ class User(AbstractBaseUser, PermissionsMixin):
     preferences = models.TextField(default=DEFAULT_USER_PREFERENCES)
     disk_space = models.FloatField(default=524288000, help_text=_('How many bytes a user can upload'))
 
-    information = JSONField(load_kwargs={'object_pairs_hook': collections.OrderedDict}, null=True)
-    content_defaults = JSONField(load_kwargs={'object_pairs_hook': collections.OrderedDict}, default=DEFAULT_CONTENT_DEFAULTS)
-    policies = JSONField(load_kwargs={'object_pairs_hook': collections.OrderedDict}, default=dict)
+    information = JSONField(null=True)
```

Comment:
My understanding is that Django's `jsonfield` already deserializes dicts automatically into a dict, hence why you don't need the `load_kwargs` field anymore.

Reply:
Yep, `load_kwargs` isn't a recognized argument